api/src/main/scala/scalalab3/lyricsengine/LyricsService.scala

```diff
@@ -31,12 +52,52 @@ trait LyricsService extends HttpService {
         }
       }
     } ~
-    path("health") {
-      get {
-        complete {
-          StatusCodes.OK
+      path("health") {
+        get {
+          complete {
+            StatusCodes.OK
+          }
+        }
+      } ~
+      path("home") {
+        get {
+          respondWithMediaType(MediaTypes.`text/html`) {
+            val homePage = Source.fromURL(this.getClass.getResource("/index.html")).mkString
```

Comment:
Why do you serve all your static data through api module?
api and client should be independent applications. You can also have multiple different clients that work with the same api.
I would recommend to serve it using some http server like nginx instead.